src/Navbar.js

```diff
@@ -74,6 +75,8 @@ class Navbar extends Component {
       navbar = <div className="navbar-fixed">{navbar}</div>;
     }
 
+    const sidenavLinks = sidenav ? sidenav : links;
```

Comment:
actually why not move this to L44?

Reply:
I moved it in the latest commit. It was here because I figured it should be next to the other logic statement (fixed) but makes no difference in the end